sklearn/metrics/classification.py

```diff
@@ -1833,11 +1833,11 @@ def hamming_loss(y_true, y_pred, labels=None, sample_weight=None):
     y_type, y_true, y_pred = _check_targets(y_true, y_pred)
     check_consistent_length(y_true, y_pred, sample_weight)
 
-    if labels is None:
-        labels = unique_labels(y_true, y_pred)
-    else:
-        labels = np.asarray(labels)
-
+    if labels is not None:
+        warnings.warn("This parameter was unused in version 0.21"
+                      " and will be removed in version 0.23 as the"
+                      " number of labels is identical"
```

Comment:
But this is unchanged

Reply:
Ok, for the first item, is this workable:  
>This fixes #10580, fixes #10582, closes #12210

Reply:
Can you tell me what the exact text should be, as in this example:  
**CHANGE:** 
>This parameter was unused in version 0.21 and will be removed in version 0.23 as the number of labels is identical

**TO:**   
>This parameter was unused in version 0.21 and will be removed in version 0.23 as the number of labels is identical

I really don't understand what needs to be changed, and @SandraMNE did not either in her PR.

Reply:
I mean "This parameter was deprecated in version 0.21 and will be removed in 0.23, as it was unused." or  "This parameter is unused. It was deprecated in version 0.21 and will be removed in 0.23."